symbolic_pymc/tensorflow/printing.py

```diff
@@ -16,23 +16,23 @@ def __init__(self, formatter, buffer, depth_index):
         self.buffer = buffer
         self.formatter = formatter
         self.depth_index = depth_index
-        self.indentation = ""
+        self.depth_tracking = ""
```

Comment:
I don't understand this renaming.  Now, it's much less clear what that variable is for/doing based on its name.

Reply:
TFlowPrinter is now setup to track depth separately. The original naming convention was returned.